scale/scheduler/models.py

```diff
@@ -106,6 +122,8 @@ def get_status(self):
             sched_dict['is_online'] = sched_info.is_online
             sched_dict['is_paused'] = sched.is_paused  # Note this must be pulled from the database
             sched_dict['hostname'] = sched_info.hostname
+            #TODO: Get resource level from somewhere else or set to GOOD until logic is developed
+            # sched_dict['resource_level'] = sched_info.resource_level
```

Comment:
You can set this using `sched.resource_level`. The line `sched = self.get_master()` above is retrieving the scheduler model.